src/main/java/thefarm/orchards/apple/model/Apple.java

```diff
@@ -0,0 +1,88 @@
+package thefarm.orchards.apple.model;
+
+public class Apple {
+    private String stem;
+    private String blossom;
```

Comment:
What kind of use would this be? Would it be for a single blossom? Or perhaps would we use it to describe a whole tree, which is outside the Apple itself?

Reply:
Blossom in this case is only a description of the bottom of the apple, perhaps I should let that one go and add "core"?

Reply:
The core of the apple is not usually consumed because it has seeds an a less desirable texture, not much flavor or juice.

Reply:
Unless there is either something unusual about a core, it is probably something considered to be negligible and standard. Perhaps, though, you could have a parent object called "fruit", which has a property called ``hasNonEdibleCore``. Apples, Mangos, and Peaches, for example, would have this property set to true as a category, rather than individual objects of each category.